### PR TITLE
i18n(zh-cn): translate reference/errors/unknown-content-collection-error.mdx , reference/errors/invalid-content-entry-frontmatter-error.mdx, reference/errors/invalid-content-entry-slug-error.mdx,reference/errors/content-schema-contains-slug-error.mdx 

### PR DESCRIPTION
**Pull request by Liqiaomiao**
#### What kind of changes does this PR include?

- Translated content

#### Description
- What does this PR change? Give us a brief description.
   translate 
   - reference/errors/unknown-content-collection-error.mdx 
   -  reference/errors/invalid-content-entry-frontmatter-error.mdx
   -  reference/errors/invalid-content-entry-slug-error.mdx
   - reference/errors/content-schema-contains-slug-error.mdx

### CODE REVIEW

**Comment by Liqiaomiao** [comment]
@liruifengv

**Comment by CaptainOfPhB** [comment]
@Liqiaomiao Please fix those issues

File: src/content/docs/zh-cn/reference/errors/content-schema-contains-slug-error.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+> **ContentSchemaContainsSlugError**: 内容集合模式不应包含 `slug`，因为它作为保留字段用于 slug 的生成。请从 Collection_Name 集合模式中删除此内容。
+
+## 哪里发生了错误？
+
+内容集合模式不应包含 `slug` 字段。这个字段是由 Astro 用于生成条目 slug 的保留字段。请从该模式中删除 slug 字段。你可以在前置信息中使用自定义的 slugs。
+
+**参见:**
```
Comment (CaptainOfPhB) [inline src/content/docs/zh-cn/reference/errors/content-schema-contains-slug-error.mdx:13]: 统一为：请参阅

File: src/content/docs/zh-cn/reference/errors/invalid-content-entry-frontmatter-error.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+title: Content entry frontmatter does not match schema.
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+> **Example error message:**<br/>
```
Comment (CaptainOfPhB) [inline src/content/docs/zh-cn/reference/errors/invalid-content-entry-frontmatter-error.mdx:7]: 翻译为“错误信息示例”

File: src/content/docs/zh-cn/reference/errors/invalid-content-entry-slug-error.mdx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+> `COLLECTION_NAME` → `ENTRY_ID` 的 slug 无效。slug 必须是一个字符串。
+
+## 哪里发生了错误？
+
+`src/content/` 中的条目具有无效的 `slug`。此字段作为保留字段，用于生成条目的 slug，并且在存在时必须是一个字符串。
+
+**参见:**
```
Comment (CaptainOfPhB) [inline src/content/docs/zh-cn/reference/errors/invalid-content-entry-slug-error.mdx:13]: 统一为：请参阅

File: src/content/docs/zh-cn/reference/errors/unknown-content-collection-error.mdx [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+## 哪里发生了错误？
+
+Astro 在加载内容集合时遇到未知错误。
+
+这可能是由 `src/content/config.ts` 文件中的某些错误引起的，或一些内部错误。
+
+如果你能够稳定的复现这个错误，请[创建一个issue](https://astro.build/issues/)，我们将不胜感激。
```
Comment (CaptainOfPhB) [inline src/content/docs/zh-cn/reference/errors/unknown-content-collection-error.mdx:13]: issue 前缺少空格

**Comment by CaptainOfPhB** [comment]
LGTM

File: src/content/docs/zh-cn/reference/errors/invalid-content-entry-frontmatter-error.mdx [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+**blog** → **post.md** 的 frontmatter 与集合模式不匹配。<br/>
+需要 "title"。<br/>
+"date" 必须是有效的日期。
+
+## 哪里发生了错误？
+
+在 `src/content/` 目录下的 Markdown 或 MDX 条目与其集合模式不匹配。
+确保所有必需的字段都存在，并且所有字段都是正确的类型。
+请检查 `src/content/config.*` 文件中的集合模式。
+更多信息，请参阅[内容集合](/zh-ch/guides/content-collections/)
```
Comment (Yan-Thomas) [inline src/content/docs/zh-cn/reference/errors/invalid-content-entry-frontmatter-error.mdx:17]:
```suggestion
更多信息，请参阅[内容集合](/zh-cn/guides/content-collections/)
```

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Welcome to Team Docs, @Liqiaomiao!

**Pull request merged**